docs/hopr-documentation/docs/developers/starting-local-cluster.md

```diff
@@ -17,26 +17,30 @@ To run a HOPR cluster, we suggest the following options to you based on your Ope
 - Windows: Gitpod
 - Linux or macOS: Gitpod or Local
 
+## Use the latest release
+Due to the rapid development done on the project, you might be better off using a stable
+release. The latest stable release known at the time of writing is [`lisbon`](https://github.com/hoprnet/hoprnet/archive/refs/heads/release/lisbon.zip).
+
 ## Gitpod Setup
 
 The simplest and fastest way to setup a HOPR cluster is using [Gitpod](https://gitpod.io). Gitpod is a cloud tool used to create
-automated dev environments in seconds. We have configured our [HOPR monorepo](https://gitpod.io/#https://github.com/hoprnet/hoprnet)
+automated dev environments in seconds. We have configured our [HOPR monorepo](https://github.com/hoprnet/hoprnet/tree/release/lisbon)
```

Comment:
shouldn't we have `gitpod.io/#` in the beginning?

Reply:
Good catch!